insights/config/specs.py

```diff
@@ -22,7 +22,8 @@
     "brctl_show"                : CommandSpec("/usr/sbin/brctl show"),
     "candlepin.log"             : First([SimpleFileSpec(r"var/log/candlepin/candlepin.log", large_content=True),
                                     SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/candlepin.log", large_content=True)]),
-    "candlepin_error_log"       : NoneGroup([SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True)]),
+    "candlepin_error_log"       : First([SimpleFileSpec(r"var/log/candlepin/error.log", large_content=True),
```

Comment:
For the master branch, we'd better modify the`insights/specs.py` together. Please refer to `getcert_list` in the file.

Reply:
I have added insights/specs.py file also. Kindly review it.